### PR TITLE
Register debarchito.is-a.dev

### PR DESCRIPTION
**Pull request by debarchito**
Register debarchito.is-a.dev with CNAME record pointing to https://debarchito.github.io.

### CODE REVIEW

File: domains/debarchito.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
+        "owner": {
+           "username": "debarchito",
+           "email": "44723456+debarchito@users.noreply.github.com",
+           "discord": "739497344780992564"
+        },
+    
+        "record": {
+            "CNAME": "https://debarchito.github.io"
```
Comment (CuteDog5695) [inline domains/debarchito.json:9]:
```suggestion
            "CNAME": "debarchito.github.io"
```

File: domains/debarchito.json [hunk 2]
```diff
@@ -0,0 +1,12 @@
+{
+        "owner": {
+           "username": "debarchito",
+           "email": "44723456+debarchito@users.noreply.github.com",
```
Comment (CuteDog5695) [inline domains/debarchito.json:4]: Either replace the noreply email with your own email or leave it blank.

**Review by iostpa — CHANGES_REQUESTED** [review]

**Review by iostpa — APPROVED** [review]
Seems good to me now

File: domains/debarchito.json (hunk 2 above)
Comment (debarchito) [reply domains/debarchito.json]: updated!

**Pull request merged**